.github/workflows/build-test-measure.yml

```diff
@@ -0,0 +1,154 @@
+          path: builds
+
+  upload-to-gcs:
+    name: Upload plugin ZIPs to Google Cloud Storage
+    runs-on: ubuntu-latest
+    needs: release-zip
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: amp-${{ needs.release-zip.outputs.branch-name }}-${{ needs.release-zip.outputs.git-sha-8 }}
+          path: builds
+
+      - name: Setup Google Cloud SDK
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCS_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCS_APPLICATION_CREDENTIALS }}
+
+      - name: Upload artifacts to bucket
+        run: gsutil cp -r builds/* gs://ampwp_github_artifacts/${{ github.ref }}
+
+  comment-on-pr:
+    name: Comment on PR with links to plugin ZIPs
+    # Only run this job if it's a PR. One way to check for that is if `github.head_ref` is not empty.
+    if: ${{ github.head_ref && github.head_ref != null }}
+    runs-on: ubuntu-latest
+    needs: upload-to-gcs
+
+    outputs:
+      pr_number: ${{ steps.get-pr-number.outputs.num }}
+      comment_body: ${{ steps.get-comment-body.outputs.body }}
+
+    steps:
+      - name: Get PR number
+        id: get-pr-number
+        run: echo "::set-output name=num::$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')"
+
+      - name: Check if a comment was already made
+        id: find-comment
+        uses: peter-evans/find-comment@v1
+        with:
+          issue-number: ${{ steps.get-pr-number.outputs.num }}
+          comment-author: github-actions[bot]
+          body-includes: Download [development build]
+
+      - name: Get comment body
+        id: get-comment-body
+        # Setting a multi-line output requires escaping line-feeds. See <https://github.community/t/set-output-truncates-multiline-strings/16852/3>.
+        run: |
+          body="Plugin builds for ${{ github.sha }} are ready :cook:!
+          - Download [development build](https://storage.googleapis.com/ampwp_github_artifacts/${{ github.ref }}/dev/amp.zip)
+          - Download [production build](https://storage.googleapis.com/ampwp_github_artifacts/${{ github.ref }}/prod/amp.zip)"
+          body="${body//$'\n'/'%0A'}"
+          echo "::set-output name=body::$body"
+
+      - name: Create comment on PR with links to plugin builds
+        if: ${{ steps.find-comment.outputs.comment-id == '' }}
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          issue-number: ${{ steps.get-pr-number.outputs.num }}
+          body: ${{ steps.get-comment-body.outputs.body }}
+
+      - name: Update comment on PR with links to plugin builds
+        if: ${{ steps.find-comment.outputs.comment-id != '' }}
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          comment-id: ${{ steps.find-comment.outputs.comment-id }}
+          edit-mode: replace
```

Comment:
Question: if the previous comment is being replaced, why do I see a new comment with the old one being shown as obsolete? 

![image](https://user-images.githubusercontent.com/134745/84105720-fe6a7180-a9cd-11ea-9fe0-66520f9c6d4b.png)

Reply:
Ah that was me marking them as outdated. The problem here is the key phrase that the job is looking for to identify the existing comment changed; the phrase `Download the development build` no longer existed, so now I'm using `Download [development build]` instead.